CONTRIBUTING.md

```diff
@@ -33,16 +39,28 @@ $ git clone git@github.com:<user>/kubebuilder.git $GOPATH/src/sigs.k8s.io/kubebu
 1. Build the project by using the command `make build` 
 1. Run the command `make install` to create a bin with the source code 
 
-## How to test kubebuilder locally
+**NOTE** In order to check the local environment run `make go-test`.
+
+## What to do before submitting a pull request 
 
-1. Run the tests by using the command `make test`. It will execute unit tests. 
```

Comment:
Leave this, you need to run tests before submitting PRs.

Reply:
It is above:

> **NOTE** In order to check the local environment run `make go-test`.

Reply:
Well, no test is mandatory then, as they are all run by the CI. It all depends on what you call mandatory.

Reply:
The make generate is not a test. It is mandatory since is a requirement keep the testdata updated based on the latest changes indeed for we are able to do the review.